csharp/test/Ice/adapterDeactivation/AllTests.cs

```diff
@@ -80,7 +80,8 @@ public static Test.ITestIntfPrx allTests(global::Test.TestHelper helper)
                     var prx = IObjectPrx.Parse("dummy:tcp -h localhost -p 12346 -t 20000:tcp -h localhost -p 12347 -t 10000", communicator);
                     adapter.SetPublishedEndpoints(prx.Endpoints);
                     test(adapter.GetPublishedEndpoints().Length == 2);
-                    test(Collections.Equals(adapter.CreateProxy(new Identity("dummy", "")).Endpoints, prx.Endpoints));
+                    test(Collections.Equals(adapter.CreateProxy(new Identity("dummy", ""), IObjectPrx.Factory).Endpoints,
```

Comment:
Should we have a second overload without factory for `IObjectPrx`?

Reply:
I don't think we should: there is no reason to favor IObjectPrx over typed proxies when creating a proxy with CreateProxy.